report_common_rpl/__manifest__.py

```diff
@@ -1,12 +1,12 @@
-# Copyright 2019 Quartile Limited
+# Copyright 2019-2022 Quartile Limited
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
-    "name": "Report Common Py3o",
-    "version": "12.0.1.0.0",
+    "name": "Report Common",
+    "version": "12.0.1.2.0",
```

Comment:
Do you have special meanings to skip the version "12.0.1.1.0"?

Reply:
It was just a typo. Fixed now.